public/content/documentation/web/component/autocomplete.md

```diff
@@ -0,0 +1,156 @@
+
+   - Swipe: Focus moves to the input, traverses list
+   - Doubletap: Opens select, chooses option
+
+3. Listen to screenreader output on all devices
+
+   - Name: Its purpose is clear
+   - Role: It identifies itself as a select, popup, menu/submenu, listbox or combobox
+   - Group: Its label is read and selected options are read
+   - State: It indicates the value of the text input
+     
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/autocomplete](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/autocomplete)
+
+## Gherkin
+
+### #a11y - Web Accessibility Acceptance Criteria
+
+How to test a header
+
+GIVEN THAT I am on a page with a header landmark
+
+1. Keyboard for mobile & desktop
+
+   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+   - WHEN I use the tab key to move focus to the text input I SEE focus is strongly visually indicated
+   - THEN when I use the arrow keys to select an option I SEE the selected option is the new text input value
+   - THEN when I use the enter key I SEE the selected option is changed and focus returns to the text input
+   - THEN when I use the escape key when the select is open I SEE it collapses and focus moves to the text input
+
+2. Desktop screenreader
+
+   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND I use the tab key to move focus to the text input
+      - I HEAR Its purpose is clear
+      - I HEAR It identifies itself as a select, popup, menu/submenu, listbox or combobox
+      - I HEAR Its label is read and selected options are read
+      - I HEAR It indicates the value of the text input
+   - THEN when I use the arrow keys to select an option I HEAR the selected option is the new text input value
+   - THEN when I use the enter key I HEAR the selected option is changed and focus returns to the text input
+   - THEN when I use the escape key when the select is open I HEAR it collapses and focus moves to the text input
+
+3. Mobile screenreader
+
+   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND I swipe to focus on a select
+      - I HEAR Its purpose is clear
+      - I HEAR It identifies itself as a select, popup, menu/submenu, listbox or combobox
+      - I HEAR Its label is read and selected options are read
+      - I HEAR It indicates the value of the text input
+   - THEN when I doubletap with the select in focus I HEAR the selected option is changed
+
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/autocomplete](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/autocomplete)
+
+## Code Examples
+
+### Custom listbox with autocomplete
+Custom listboxes are notoriously difficult to build in an accessible fashion for screenreaders.
+
+- [WCAG Examples](https://w3c.github.io/aria-practices/examples/combobox/combobox-autocomplete-list.html)
```

Comment:
Following this link, it says the content has moved. Let's update the link to:
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/
We could also label the link with the H1 on the page it directs to, which would be "Editable Combobox With List Autocomplete Example" - but not a requirement!